### PR TITLE
[frontend] chore: add script to add error codes as translation keys

### PR DESCRIPTION
**Pull request by Kroustille**
# Context
> Briefly describe the purpose and context of this PR. Include relevant screenshots or screen recordings for the product team.

This PR adds a script to automatically add error codes from API to frontend translation keys. It also deletes error codes that are not used anymore.
This way, we're sure every error is translated for our users.

This script does not modify `fr.json` because it's already updated by our auto translation script.

- add a new error in `error.code.ts` file
- delete an error in `error.code.ts` file
- launch script
- a new error is added and another is deleted in `en.json`

## Related issues

- Related to #976

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [x] I tested all features and edge cases
- [x] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-front/scripts/extract-error-code-translation-keys.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-console */
```
Comment (jpkha) [inline apps/portal-front/scripts/extract-error-code-translation-keys.ts:1]: Ignore from the eslint config than adding a comment here

**Review by jpkha — APPROVED** [review]
💯
